### PR TITLE
Added missing quotes info screeen

### PR DESCRIPTION
**Pull request by sladan-hedvig**
## [APP-XXX]

- Handled from help centar - with or without select insurance screen
- Handled from insurance, edit insurance, upgrade coverage
- Handled from termination flow

From edit insurance:
simulator_screenshot_56E2DA58-9CB1-4DD8-9BC3-8C505B50BA8E

From termination flow:
simulator_screenshot_CAE6FE46-98FF-4116-9E52-9F4628F1FB41

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ public class ChangeTierNavigationViewModel: ObservableObject {
     @Published public var isCompareTiersPresented = false
     @Published public var isInsurableLimitPresented: InsurableLimits?
     @Published public var document: Document?
-
+    let useOwnNavigation: Bool
     let router: Router
 
     //make sure to set it!!!
```
Comment (juliaendre) [inline Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift:16]: can we remove this comment?

File: Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift [hunk 2]
```diff
@@ -41,13 +42,24 @@ public class ChangeTierNavigationViewModel: ObservableObject {
...
+        self.useOwnNavigation = true
     }
 
     public static func getTiers(input: ChangeTierInputData) async throws -> ChangeTierIntentModel {
         let client: ChangeTierClient = Dependencies.shared.resolve()
         let data = try await client.getTier(input: input)
         return data
     }
+
+    func missingQuotesGoBackPressed() {
```
Comment (juliaendre) [inline Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift:54]: Is this not the same as?

```
    func missingQuotesGoBackPressed() {
        if useOwnNavigation {
            router.dismiss()
        } else {
            router.pop()
        }
    }
```

I think that would be cleaner and I don't see the purpose of checking `changeTierContractsInput `

**Review by juliaendre — CHANGES_REQUESTED** [review]

File: Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift:16]: This is like info, can we keep it?

Comment (juliaendre) [reply Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift:16]: Ah yes but then I think we should rephrase it to look more professional. Maybe "Note: important to set this." or something like that

File: Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift (hunk 2 above)
Comment (sladan-hedvig) [reply Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift:54]: Not really since we could use own navigation and still have changeTierContractsInput

Comment (sladan-hedvig) [reply Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift:54]: But yeah, can be merged 👍

**Review by juliaendre — APPROVED** [review]
